pywbemtools/pywbemcli/_display_cimobjects.py

```diff
@@ -105,14 +110,21 @@ def display_cim_objects(context, cim_objects, output_format, summary=False,
         If False, strings are not encased by quote marks in the table view for
         instance displays. The default is True so that strings are encased in
         quotes in all views.
+
+      ignore_empty_properties (:class:`py.bool`):
+        If True, cases where a property has None value in all of the instances
+        will not be included in table output. They will be included in CIM
+        object(MOF) output. If False, The properties that do not have a value
+        will be included in tables.  This allows table output to ignore
+        properties that do not add value to the table display
```

Comment:
Can we rename this parameter to `ignore_null_properties`, because the notion of "empty" can also apply to array properties or string properties with an empty string.

Reply:
DONE